trin-utils/Cargo.toml

```diff
@@ -12,7 +12,12 @@ authors = ["https://github.com/ethereum/trin/graphs/contributors"]
 build = "build.rs"
 
 [dependencies]
+anyhow = "1.0.68"
+directories = "3.0"
+discv5 = { version = "0.2.1", features = ["serde"]}
+ethereum-types = "0.12.1"
```

Comment:
Yeah, this one falls in a bit of a grey area. The convention I was following for `trin-types` / `trin-utils` was that they contain types / utils that are "shared between workspace members". Since these are used by `portalnet` and also inside the main `src/lib.rs` ... I interpreted this as being shared between multiple workspace members. Even though just as easily, we can import them from `portalnet` inside `src/lib.rs`. So, I moved these utils back to `portalnet/utils` since that seems reasonable enough to me